lib/mods/replace-window-functions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  formatReplacement,
+  getOriginal,
+} = require('../originals-helper.js');
+
+/**
+ * @param {*} node - the node to examine
+ *
+ * @return {boolean} whether the node represents a window function
+ */
+function isWindowProperty(node) {
+  const symbol = node.getExpression().getSymbol();
+  const firstDeclaration = (symbol) ? symbol.getDeclarations()[0] : undefined;
+
+  // TODO: may require a deeper check
+  // e.g. setTimeout web returns number, Node returns Timer
+  if (firstDeclaration) {
+    return window.hasOwnProperty(firstDeclaration.getName());
```

Comment:
What is `window` here? `window` shouldn't exist in Node.js... Maybe the testing environment is introducing it!? That'd be bad because it'd fail outside of our tests.

Reply:
I wasn't really sure what this was or how it got here either but it seemed to work.

I could look into alternatives for this if we think it's gonna fail outside of the tests (which I thought were running in a node environment but maybe that's a jest config problem)

Reply:
Yeah, I think that's a Jest config problem. You can try by running the command from the readme (`npm run rewrite ...`).

Reply:
So I swapped this here, but I have yet to run the tests in a node environment because running jscodeshift is broken - will be fixed as I resolve #30 in an upcoming PR